hw_9/functional.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[115]:
+
+
+import sys
+
+
+# In[59]:
+
+
+# sequential_map
+def sequential_map(*args):
+    container = args[-1]
+    for i in args[:len(args) - 1]:
+        container = list(map(i, container))
+    return(container)
+
+
+# In[61]:
+
+
+# consensus_filter
+def consensus_filter(*args):
+    container = args[-1]
+    for i in args[:len(args) - 1]:
+        container = list(filter(i, container))
+    return(container)
+
+
+# In[64]:
+
+
+# conditional_reduce
+def reduce(func, cont):
+    x = cont[0]
+    for i in cont[1::]:
+        x = func(x, i)
+    return(x)
+def conditional_reduce(func1, func2, container):
```

Comment:
Отступ в две строки между функциями
```suggestion
    return(x)
    
    
def conditional_reduce(func1, func2, container):
```